tests/rewriter_test.py

```diff
@@ -14,10 +14,10 @@
 def rewrite_and_compare(prog: str, expected_prog: str,
                         transformation: Callable[[ModuleOp, Rewriter], None]):
     ctx = MLContext()
-    builtin = Builtin(ctx)
-    arith = Arith(ctx)
-    scf = Scf(ctx)
-    func = Func(ctx)
+    builtin = Builtin(ctx)  # noqa
+    arith = Arith(ctx)  # noqa
+    scf = Scf(ctx)  # noqa
+    func = Func(ctx)  # noqa
```

Comment:
I'm assuming that this is here to make a linter not look at these lines, such that it does not generate `unused variable` errors. IMO, this is something we have to change in the future, so I would not add `noqa` comments. How about  using `_ = Func(ctx)` instead?

Reply:
Yes, we can use `_ = Func(ctx)`, or `Func(ctx)` directly.